### PR TITLE
Update Set-CsWebServiceConfiguration.md

### PR DESCRIPTION
**Pull request by priyanshamishra**
Adding clarification for the parameter -EnableCORS.

### CODE REVIEW

**Comment by priyanshamishra** [comment]
@MicrosoftDocs/marveldocs-pubops PR is ready for review. Please review and publish.

**Comment by Kateyanne** [comment]
@get-itips please review @JohanFreelancer9 please edit. Thank you. @e0i fyi

**Review by get-itips — CHANGES_REQUESTED** [review]
The parameter needs to be also added to the SYNTAX section of the markdown file.

**Review by dariomws — APPROVED** [review]
I'm ok once @get-itips's suggestion is applied.

File: skype/skype-ps/skype/Set-CsWebServiceConfiguration.md [hunk 1]
```diff
@@ -287,6 +287,18 @@ Default value: None
...
+
+If set to True, Cross-Origin Resource Sharing is enabled.
+
+```yaml
+Type: Boolean
+Position: Named
+Default value: False
+Accept pipeline input: False
+Accept wildcard characters: False
+Applies to: Lync Server 2013, Skype for Business Server 2015, Skype for Business Server 2019
```
Comment (dariomws) [inline skype/skype-ps/skype/Set-CsWebServiceConfiguration.md:300]:
```suggestion
Type: Boolean
Parameter Sets: (All)
Aliases: 
Applicable: Lync Server 2013, Skype for Business Server 2015, Skype for Business Server 2019

Required: False
Position: Named
Default value: False
Accept pipeline input: False
Accept wildcard characters: False
```

**Review by dariomws — COMMENTED** [review]
Sorry, I found a small detail

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@priyanshamishra please review and update

**Comment by dstrome** [comment]
@priyanshamishra @Kateyanne - I fixed the syntax issue that was causing the error, however, there are some parameter properties missing from the new parameter that was added:

- Parameter Sets
- Aliases
- Required

Also, the new parameter doesn't appear in any of the parameter sets at the top of the topic. 

All of these issues must be fixed before you publish the topic, so I've placed the *Do not merge* label on the PR. When these issues are fixed, you can remove the label and merge the PR.

File: skype/skype-ps/skype/Set-CsWebServiceConfiguration.md [hunk 2]
```diff
@@ -287,6 +287,18 @@ Default value: None
...
+
+If set to True, Cross-Origin Resource Sharing is enabled.
+
+```yaml
+Type: Boolean
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+Applicable: Lync Server 2013, Skype for Business Server 2015, Skype for Business Server 2019
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsWebServiceConfiguration.md:300]:
```suggestion
Position: Named
Parameter Sets: (All)
Aliases:
Applicable: Lync Server 2013, Skype for Business Server 2015, Skype for Business Server 2019

Required: False
Position: Named
Default value: False
Accept pipeline input: False
Accept wildcard characters: False
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@priyanshamishra - I have added the missing parameter properties as suggested changes - please commit the suggestions if you agree with them. Also, please add the new parameter in the parameter sets at the top of the topic as per @dstrome's request.

**Comment by yogkumgit** [comment]
@JohanFreelancer9  and @get-itips

**Comment by get-itips** [comment]
> @JohanFreelancer9 and @get-itips

The PR author still needs to apply suggestions, for example, include the parameter in the syntax section.

**Comment by yogkumgit** [comment]
@JohanFreelancer9 and @get-itips

**Comment by get-itips** [comment]
@priyanshamishra The parameter -EnableCORS is still missing from the syntax section, can you add it please? Thanks

**Comment by priyanshamishra** [comment]
> @priyanshamishra The parameter -EnableCORS is still missing from the syntax section, can you add it please? Thanks

I have added the parameter to the syntax section.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips

**Review by get-itips — APPROVED** [review]

**Comment by Kateyanne** [comment]
@dstrome do you concur this can now be merged?

**Pull request merged**